src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -703,15 +703,15 @@ namespace
         {
             const string widget_base_key = prefix + ".bounces.";
 
-            QSpinBox* max_bounces = create_integer_input(widget_base_key + "max_bounces", 0, 10000, 1);
+            QSpinBox* max_bounces = create_integer_input(widget_base_key + "max_bounces", 0, 100, 1);
```

Comment:
Would it make sense to make this a constant and reference it everywhere.
Would be useful if we need to adjust the max bounces limit in the future.